server/service/src/document/patient/update.rs

```diff
@@ -37,8 +38,10 @@ pub fn update_patient(
     let patient = ctx
         .connection
         .transaction_sync(|_| {
-            let patient = validate(ctx, service_provider, &store_id, &input)?;
-            let doc = generate(user_id, &patient, input)?;
+            let patient_schema = validate(ctx, service_provider, &store_id, &input)?;
+            let patient_id = patient_schema.id.clone();
```

Comment:
oh? the patient_id is the schema_id? what's this all about?
perhaps I'm misunderstanding something

Reply:
Yes confusing naming, patient_schema here is a patient object in the shape as specified in the json schema. I rename it back to just patient, thanks